docs/organizations/roles-permissions.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Roles and Permissions
+description: Learn about how to create and manage roles and permissions for Clerk Organizations and their members.
+---
+
+# Roles and Permissions
+
+Clerk supports modeling your own custom Roles and Permissions to control access to resources within your application when you use [Organizations](/docs/organizations/overview).
+
+## Roles
+
+Roles determine a user's level of access to the Organization. You can extend a Roles privileges and access to resources by adding [Permissions](#permissions).
+
+### Default Roles
+
+For each instance there are currently two default roles:
+
+- **Admin (`org:admin`)** Offers full access to Organization resources. Members with the admin role have all the [System Permissions](#system-permissions). They can fully manage the Organization and Organization memberships.
+- **Member (`org:member`)** Offers limited access to Organization resources. Access to Organization resources is limited to only Read members Permission by default. They cannot manage the Organization and Organization memberships but can view information about other members in it.
+
+<Callout type="info">
+If you enabled Organizations for your Application before December 2023, the Admin Role is `admin` and the Member Role is `basic_member`, instead of `org:admin` and `org:member`
+</Callout>
+
+### Custom Roles
+
+You can create up to 10 of custom Organization Roles per application instance to meet your application needs. If you need more than 10 Roles, reach out to [support@clerk.dev](mailto:support@clerk.dev).
+
+Custom Roles can be granted Permissions and access. For example you can create a new Role of “billing” (`org:billing`) which can be used to group users who belong to a specific department of the Organization and have permission to manage credit card information, invoices, and other resources related to billing.
+
+### The Creator Role
+
+When a user creates a new Organization, that user is automatically added as the Organization's first member and assigned the Creator Role. By default, Admin is the Creator Role.
+
+<Callout type="warning">
+You can't delete a Role if it's used as the Organization Creator Role. However, you can [reassign the Creator Role to another role](/docs/organizations/creator-role).
+</Callout>
```

Comment:
What happens when you delete a `Role` with users that isn't used as Creator Role?

Reply:
Good question. You can't. You can only delete Roles that have no assignees... so you'd need to migrate everyone to a new one, then delete it.

Reply:
Ooooh Interesting! Thats what I was assuming based off what I read and the `Creator Role` information.